translations/pl.ts

```diff
@@ -1015,7 +1015,7 @@
     <message>
         <source>Waypoint</source>
         <extracomment>search page, entry for list near waypoints from collections</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>strona wyszukiwania, wpis na liście w pobliżu punktów trasy z kolekcji </translation>
```

Comment:
please translate souce: "Waypoint" not extracomment ;-)

Reply:
sorry for that. fixed